### DISCUSSION TITLE
Basic Ctrl+F for the in-client browser

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#22014] [Ideas]
This'd make the new osu!lazer migration guide FAQ easier to navigate in the client's inbuilt browser, for example.

**Comment by ghost** [comment]
I attempted this but it seems very difficult to figure out where to scroll to because you have to know where exactly the text is on the page. Highlighting text doesn't seem trivial either.

**Comment by bdach** [comment]
this is 99.9999% not happening, sorry.

**Comment by peppy** [comment]
The is a plan for a global search mode, similar to osu-web. It needs to be implemented though.
